base/random.jl

```diff
@@ -257,13 +257,19 @@ globalRNG() = GLOBAL_RNG
 Pick a random element or array of random elements from the set of values specified by `S`; `S` can be
 
 * an indexable collection (for example `1:n` or `['x','y','z']`), or
-* a `Dict`, a `Set` or an `IntSet`, or
+* an `AbstractString`, a `Dict`, a `Set` or an `IntSet`, or
 * a type: the set of values to pick from is then equivalent to `typemin(S):typemax(S)` for
   integers (this is not applicable to [`BigInt`](@ref)), and to ``[0, 1)`` for floating
   point numbers;
 
 `S` defaults to [`Float64`](@ref).
 
+Note that the complexity of `rand(rng, s::AbstractString)` is linear
```

Comment:
Not really necessary, but this could be a Documenter `note` block, like
```
!!! note
    The complexity of `rand(rng, s::AbstractString)` is linear in the
    length of `s` if `s` is not of type `String`. If called more than
    a few times, use `rand(rng, collect(s))` instead.
```